### PR TITLE
feat: parse compounds from manifest in aliquots

### PR DESCRIPTION
**Pull request by aota001**
* Add `compounds` to Well attributes in container.py
* Add feature to append `compounds` in harness.py
* Update all tests

### CODE REVIEW

**Comment by aota001** [comment]
This is still wip, but wanted to start sharing so that I can make corrections as I go along since I'm making changes in containers... thanks!

File: autoprotocol/container.py [hunk 1]
```diff
@@ -168,6 +227,25 @@ def set_name(self, name):
         self.name = name
         return self
 
+    def set_compounds(self, compounds):
+        """
+            Set a name for this well for it to be included in a protocol's
```
Comment (joshhacksthings) [inline autoprotocol/container.py:232]: nit: "Set a compound" not "name"

File: autoprotocol/container.py [hunk 2]
```diff
@@ -207,7 +285,7 @@ def __repr__(self):
         Return a string representation of a Well.
 
         """
-        return f"Well({str(self.container)}, {str(self.index)}, " f"{str(self.volume)})"
+        return f"Well({str(self.container)}, {str(self.index)}, " f"{str(self.volume)}, {str(self.compounds)})"
```
Comment (joshhacksthings) [inline autoprotocol/container.py:288]: is this something we'll want to return if `.humanize()` is called on the well?

**Comment by aota001** [comment]
Thanks @joshhacksthings. I just put up ASC proposal in https://github.com/autoprotocol/ascs/pull/28. Will add this to the description once it is approved.

**Comment by aota001** [comment]
Will close this PR until the parent ASC is resolved. Also, due to a change in the product requirement, I no longer need this feature at this moment.

**Pull request closed without merging**